index.html

```diff
@@ -16,16 +17,123 @@
 </head>
 
 <body>
-  <div class="container">
+  <div class="container-fluid d-flex justify-content-center sticky-top nav-bg">
```

Comment:
Here you would want "container-fluid sticky top" to go before a container. That way it will wrap a whole container and the container will stick before sticking after carousel.